src/shared/db/mongodb/mongo-manager.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+require('dotenv').config()
+
+
+const openMongoConnection = () => {
+    const db = mongoose.connection;
+    db.on('error', console.error.bind(console, 'connection error:'));
+    db.once('open', function callback () {
+        console.log("connected to MongoDB");
+    });
+    mongoose.connect('mongodb+srv://courtneyagregg:GodComplex11@mod5.nsqu7m2.mongodb.net/?retryWrites=true&w=majority&appName=Mod5');
```

Comment:
You would ideally place the actual connection string to be a reference to your .env file behind the scene. Something like process.env.ATLAS_URI instead of the actual string would be best.

Reply:
Will do!